views.py

```diff
@@ -1,27 +1,26 @@
 from django.shortcuts import render, redirect
-from django.core.urlresolvers import reverse
+from django.urls import reverse
 
 from plugins.annotators import plugin_settings
 from utils import setting_handler
 
-def index(request):
-    plugin = plugin_settings.get_self()
 
-    enable_annotran = setting_handler.get_plugin_setting(plugin, 'enable_annotran', request.journal, create=True,
-                                                         pretty='Enable Annotran')
+def index(request):
+    plugin = plugin_settings.AnnotatorsPlugin.get_self()
 
-    enable_hypothesis = setting_handler.get_plugin_setting(plugin, 'enable_hypothesis', request.journal, create=True,
-                                                           pretty='Enable Hypothesis')
+    setting_group_name = 'plugin:annotators'
+    enable_hypothesis = setting_handler.get_setting(
```

Comment:
This should use `setting_handler.get_plugin_setting` to avoid duplicating the logic here